dart_assignment1/lib/component/manufactured_item.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:dart_assignment1/component/item_factory.dart';
+import 'Item.dart';
+
+class ManufacturedItem extends Item {
+    late double mtax;
```

Comment:
why do we require an extra member for tax here ?

Reply:
Its like a temporary variable to store the results and then assign it to `tax' which is actually a getter to the abstract class Item. to set the value of private variable _tax